### DISCUSSION TITLE
TNT Series

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#345] [Show and tell]
### Channel Name

TNT Series

### Channel ID

TNTSeriesMexico.mx

### Stream URL

https://edge2-ccast-sl.cvattv.com.ar/live/c3eds/TNTSeries/SA_SAGEMCOM/TNTSeries.m3u8

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by freearhey** [comment]
<img width="759" alt="image" src="https://user-images.githubusercontent.com/7253922/158994780-2e373fd5-0f74-4e4c-be7e-ecdd539a9f20.png">

**Comment by R4IN2007** [comment]
https://cdn.prepro.cvattv.com.ar/live/vod/TNTSeries/SA_Live_WebVTT_FTA/TNTSeries.m3u8

**Comment by freearhey** [comment]
https://cdn.prepro.cvattv.com.ar/live/vod/TNTSeries/SA_Live_WebVTT_FTA/TNTSeries.m3u8 (offline)
<img width="646" alt="image" src="https://user-images.githubusercontent.com/7253922/232225773-61ab216e-fb62-42b4-8792-f710361d1867.png">
